src/compas/geometry/transformations/rotation.py

```diff
@@ -72,6 +72,9 @@ def __init__(self, matrix=None):
                 raise ValueError('This is not a proper rotation matrix.')
         super(Rotation, self).__init__(matrix=matrix)
 
+    # def __repr__(self):
+    #     return "Rotation({0})".format(self.matrix)
```

Comment:
why is this commented out?

Reply:
it doesn't work in all cases because the init of rotation checks for a true/pure rotation matrix. so if you specify a base point for the rotation it fails...